crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -1410,6 +1408,19 @@ impl CumulocityConverter {
         Ok(messages)
     }
 
+    pub fn load_and_create_supported_operations_messages(
+        &mut self,
+        external_id: &str,
+    ) -> Result<MqttMessage, ConversionError> {
+        self.supported_operations
+            .load_all(external_id, &self.config.bridge_config)?;
```

Comment:
Curious. Why do we need the bridge config to load supported operations?

Reply:
It seems just to get `c8y_prefix` eventually. Since the usage is very nested, so if you want refactoring, better to do in a separated PR.